install_msw.sh

```diff
@@ -127,6 +133,38 @@ then
     git config --global core.fileMode false
 fi
 
+if [ "WSL" = "$platform" ]
+then
+    # Install/update packages.
+    packages_list="autoconf automake bsdtar dos2unix doxygen \
+      gdb git libtool make patch pkg-config rsync unzip wget \
+      zip zsh g++-mingw-w64-i686"
+
+    # Disable shellcheck warning: Double quote to prevent globbing and word splitting.
```

Comment:
```suggestion
    # Disable shellcheck warning about the need to double quote $packages_list:
    # it can't be done here and we really want word splitting to happen here.
```